README.md

```diff
@@ -1,14 +1,13 @@
 # Maslow
 
-Maslow is a tool to create and manage needs. It's a frontend for the [Need API](https://github.com/alphagov/govuk_need_api). It's a Rails app which is part of the GOV.UK Publishing Platform.
+Maslow is a tool to create and manage needs. It's a Rails app which is
+part of the GOV.UK Publishing Platform.
 
 ## Dependencies
 
 - Ruby (1.9.3)
```

Comment:
Good spot, I've just removed it 1949a5b so hopefully it will never be out of date again :)